examples/dcosattributes/dcos1.8.4.attributes.json

```diff
@@ -0,0 +1,45 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "DCOS184"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "xtophmaster",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "xtentprivate",
+        "count": 3,
+        "vmSize": "Standard_D2_v2",
+        "attributes" : {

```

Comment:
additionally, add validation to block adding this for other orchestrator types until we can plumb that through.